Source/Fuse.Share/ios/iOSShareImpl.uno

```diff
@@ -2,24 +2,63 @@ using Uno.Compiler.ExportTargetInterop;
 using Uno.Threading;
 namespace Fuse.Share
 {
-	public extern(iOS) class iOSShareImpl
+	internal extern(iOS) class iOSShareImpl
 	{
-		[Foreign(Language.ObjC)]
 		public static void ShareText(string text, string description)
+		{
+			if (IsIPad())
+				Fuse.Diagnostics.UserWarning("iPad requires a position as the spawn origion of the Share popover", text + " - " + description);
```

Comment:
I don't think we need a warning. Just pick a reasonable default position. It's not a warning since the default is intended then.